build/javascript.mk

```diff
@@ -2,7 +2,7 @@
 # https://github.com/kripken/emscripten/blob/master/src/settings.js
 javascript-node: cflags += -DARCH_JS -D'ARCH=\"JS\"' -D MAX_STRING=128000
 javascript-node: ldflags += -s WASM=0 \
-	--memory-init-file 1
+	-s MEM_INIT_METHOD=1
```

Comment:
is this https://github.com/emscripten-core/emscripten/issues/5108 okay already?

Reply:
Didn't see that issue before, but I can confirm that with emcc version 1.38.39, switching between `--memory-init-file 1` and `-s MEM_INIT_METHOD=1` produces identical results (binary identical .js and .js.mem memory initialization file).
But I also see that using `-s MEM_INIT_METHOD=0` for asm.js builds don't work as expected (do work on Wasm builds though), unlike `--memory-init-file 0` that do. So I'll change back to using the latter as it seems more reliable.